test/esm-loader.js

```diff
@@ -19,6 +19,7 @@ test('xdm (ESM loader)', async function (t) {
     Content = await import('./context/esm-loader.mdx')
   } catch (error) {
     if (error.code === 'ERR_UNKNOWN_FILE_EXTENSION') {
+      await fs.unlink(path.join(base, 'esm-loader.mdx'))
```

Comment:
this change appears to be unrelated? What does it do?

Reply:
Yeah, was a small fix to the previous commit (on main) that I ran into again.
The esm loader requires args passed to node. This error is thrown without them. I typically do 'node some-test.js' to run a subset of the tests, where I forget that that arg is needed. And this line ensures the tests are cleaned up in that case!

Reply:
gotcha